server/app/assets/javascripts/admin_programs.ts

```diff
@@ -61,6 +77,44 @@ class AdminPrograms {
       lastUpdatedMillis,
     }
   }
+
+  static async tryCopyToClipboard(programLink: string): Promise<boolean> {
+    if (!window.navigator['clipboard']) {
+      return false
+    }
+    try {
+      ;(await window.navigator['clipboard'].writeText(
+        programLink
+      )) as unknown as Promise<void>
```

Comment:
question (blocking): we're not using the return type, why cast the return value?